ledger/participant-integration-api/src/main/scala/platform/store/backend/oracle/OracleTable.scala

```diff
@@ -51,18 +45,18 @@ private[oracle] object OracleTable {
       field.selectFieldExpression("?")
     }
     val keyFieldName = fields(keyFieldIndex)._1
-    val keyFieldSelectExpression = fields(keyFieldIndex)._2.selectFieldExpression("?")
-    s"""MERGE INTO $tableName USING DUAL on ($keyFieldName = $keyFieldSelectExpression)
-       |WHEN NOT MATCHED THEN INSERT ($tableFields)
-       |VALUES ($selectFields)
+    s"""
+       |INSERT /*+ IGNORE_ROW_ON_DUPKEY_INDEX ( $tableName ( $keyFieldName ) ) */ INTO $tableName
```

Comment:
crazy Oracleification courtesy of @matthiasS-da :bow: 
this alternative simply not exposing the errors as inserting. the impediment which @matthiasS-da figured - result counts can be incorrect - do not apply here

Reply:
Is this Oracle's way of saying `on conflict do nothing`?

Reply:
seems that way :man_facepalming:

Reply:
I am surprised you don't have to say `please` at the end of that commented hint :trollface: